diff -Nur -x 'debian*' -x '*~' pgpool-II-2.2.5/pgpool.conf.sample pgpool-II-2.2.5.new/pgpool.conf.sample
--- pgpool-II-2.2.5/pgpool.conf.sample	2009-02-15 06:26:28.000000000 +0100
+++ pgpool-II-2.2.5.new/pgpool.conf.sample	2009-10-12 15:05:05.000000000 +0200
@@ -7,21 +7,21 @@
 listen_addresses = 'localhost'
 
 # Port number for pgpool
-port = 9999
+port = 5433
 
 # Port number for pgpool communication manager
 pcp_port = 9898
 
 # Unix domain socket path.  (The Debian package defaults to
 # /var/run/postgresql.)
-socket_dir = '/tmp'
+socket_dir = '/var/run/postgresql'
 
 # Unix domain socket path for pgpool communication manager.
 # (Debian package defaults to /var/run/postgresql)
-pcp_socket_dir = '/tmp'
+pcp_socket_dir = '/var/run/postgresql'
 
 # Unix domain socket path for the backend. Debian package defaults to /var/run/postgresql!
-backend_socket_dir = '/tmp'
+backend_socket_dir = '/var/run/postgresql'
 
 # pgpool communication manager timeout. 0 means no timeout, but strongly not recommended!
 pcp_timeout = 10
@@ -54,10 +54,10 @@
 authentication_timeout = 60
 
 # Logging directory
-logdir = '/tmp'
+logdir = '/var/run/postgresql'
 
 # pid file name
-pid_file_name = '/var/run/pgpool/pgpool.pid'
+pid_file_name = '/var/run/postgresql/pgpool.pid'
 
 # Replication mode
 replication_mode = false
diff -Nur -x 'debian*' -x '*~' pgpool-II-2.2.5/pool.h pgpool-II-2.2.5.new/pool.h
--- pgpool-II-2.2.5/pool.h	2009-09-06 05:52:12.000000000 +0200
+++ pgpool-II-2.2.5.new/pool.h	2009-10-12 15:04:28.000000000 +0200
@@ -50,10 +50,10 @@
 #define HBA_CONF_FILE_NAME "pool_hba.conf"
 
 /* pid file directory */
-#define DEFAULT_LOGDIR "/tmp"
+#define DEFAULT_LOGDIR "/var/run/postgresql"
 
 /* Unix domain socket directory */
-#define DEFAULT_SOCKET_DIR "/tmp"
+#define DEFAULT_SOCKET_DIR "/var/run/postgresql"
 
 /* pid file name */
 #define DEFAULT_PID_FILE_NAME "/var/run/pgpool/pgpool.pid"
